### PR TITLE
add my list of books/movies/music page

### PR DESCRIPTION
**Pull request by 44sisi**

### CODE REVIEW

File: be/middleware/validate.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Request, Response, NextFunction } from 'express';
+
+const validateCategory = (req: Request, res: Response, next: NextFunction) => {
+  const CATEGORY_LIST = ['book', 'movie', 'music'];
+
+  let category = req.query.category as string;
```
Comment (4wangyu) [inline be/middleware/validate.ts:6]: Let's use `const` here since it doesn't get reassigned.

File: fe/src/components/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import classNames from 'classnames';
+import { NavLink, useLocation } from 'react-router-dom';
+import './pagination.scss';
+
+const pagesToDislay = 9;
+
+interface getPagesArgType {
```
Comment (4wangyu) [inline fe/src/components/Pagination.tsx:8]: Interface name should use `noun`.

File: be/middleware/validate.ts (hunk 1 above)
Comment (44sisi) [reply be/middleware/validate.ts]: done

File: be/modules/mine/mine.repo.ts [hunk 3]
```diff
@@ -1,66 +1,70 @@
...
 type ItemType = Book | MyMovie | Music;
 
-async function selectTotal(
-  category: string,
-  email: string
-): Promise<{ total: number }> {
-  const table = `user_${category}`;
+async function selectTotal(category: string, email: string): Promise<number> {
+  const uc = `user_${category}`;
+  const c_id = `${category}_id`;
```
[the remark this answers is not in the document]
Comment (44sisi) [reply be/modules/mine/mine.repo.ts]: done

File: fe/src/components/Pagination.tsx (hunk 2 above)
Comment (44sisi) [reply fe/src/components/Pagination.tsx]: done

**Pull request merged**